### DISCUSSION TITLE
Slider Component - move thumbs past each other?

### DISCUSSION THREAD

**Discussion by joncoronel** [adobe/react-spectrum#6458] [Q&A]
Using the slider component to create a range slider with multiple thumbs. Creating a linear gradient stop position slider essentially. But wondering if there's a way to move thumbs past each other. Currently, it seems thumbs aren't able to be moved past the position of another thumb.

**Comment by snowystinger** [comment]
More info here https://github.com/adobe/react-spectrum/issues/6408
